cmake/find_metasmt.cmake

```diff
@@ -0,0 +1,79 @@
+find_package(metaSMT REQUIRED CONFIG)
+message(STATUS "metaSMT_DIR: ${metaSMT_DIR}")
+list(APPEND KLEE_COMPONENT_EXTRA_INCLUDE_DIRS
+  "${metaSMT_INCLUDE_DIR}" ${metaSMT_INCLUDE_DIRS})
+# FIXME: We should test linking
+list(APPEND KLEE_SOLVER_LIBRARIES ${metaSMT_LIBRARIES})
+
+# THIS IS HORRIBLE. The ${metaSMT_CXXFLAGS} variable
```

Comment:
Does TravisCI use a specific version of metaSMT? I'm concerned about removing this code for older metaSMT versions.